packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneDataStores.spec.ts

```diff
@@ -874,7 +874,7 @@ describeNoCompat("GC data store tombstone tests", (getTestObjectProvider) => {
 				},
 				{
 					eventName:
-						"fluid:telemetry:ContainerRuntime:GarbageCollector:GC_Tombstone_Datastore_Revived",
+						"fluid:telemetry:ContainerRuntime:GarbageCollector:GC_Tombstone_DataStore_Revived",
```

Comment:
Ummm how did this work before?

Reply:
It was `Datastore` before this change. I corrected it to `DataStore` but maybe the check isn't case sensitive?

Reply:
Oh I see, yeah all good.  I'm sure it's case sensitive.